editor/gui/editor_bottom_panel.cpp

```diff
@@ -44,6 +44,18 @@
 
 void EditorBottomPanel::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE: {
+			SplitContainer *center_split = Object::cast_to<SplitContainer>(get_parent());
+			ERR_FAIL_NULL(center_split);
+			center_split->connect(SNAME("drag_ended"), callable_mp(this, &EditorBottomPanel::_center_split_drag_ended));
+		} break;
+
+		case NOTIFICATION_EXIT_TREE: {
+			SplitContainer *center_split = Object::cast_to<SplitContainer>(get_parent());
+			ERR_FAIL_NULL(center_split);
+			center_split->disconnect(SNAME("drag_ended"), callable_mp(this, &EditorBottomPanel::_center_split_drag_ended));
```

Comment:
Why are you disconnecting the signal?

Reply:
Restoring the state as it was before the node entered the tree since i connect the signal on `NOTIFICATION_ENTER_TREE` i thought it'd be a good idea to disconnect it in `NOTIFICATION_EXIT_TREE`.